cmd/dequeuer/main.go

```diff
@@ -242,21 +242,12 @@ func main() {
 }
 
 func exit(log *zap.SugaredLogger, err error, wrapStrs ...string) {
```

Comment:
Is it potentially misleading that this function can be called without the program then exiting (since it's called `exit()`)? This would happen if `err` is nil or it it's not nil, but is not printed.

We could consider renaming this function, or exiting whether or not an error is passed in (e.g. exit with code 0 if no error is passed in, exit with code 1 if it's a no-print error)

Reply:
It exits with log.falal

Reply:
Yes, but only `if err != nil && !errors.IsNoPrint(err)`

Reply:
fixed